package.json

```diff
@@ -24,7 +24,8 @@
     "react-scroll": "^1.5.2",
     "react-table": "^6.0.5",
     "sinon": "^2.3.2",
-    "universal-cookie": "^2.0.7"
+    "universal-cookie": "^2.0.7",
+    "yarn": "^0.27.5"
```

Comment:
Typically, package managers are not added as a dependency.

To remove this properly (and correctly change `yarn.lock`), please run:
`yarn remove yarn` within the project's root folder. Also, remove the `package-lock.json` please.